### DISCUSSION TITLE
none of the demos work on a 3090

### DISCUSSION THREAD

**Discussion by emersonium** [THUDM/CogVLM#405] [General]
was able to downgrade transformers and get the CLI to work, the gradio app still not working for 4 bit quantization.

**Comment by S-15-77** [comment]
@emersonium 
Try to fork the whole repo and install the req docs in requirement.txt, It works fine for me in colab

**Comment by emersonium** [comment]
thanks, all good, was able to do that, and then  rewrite the webdemo to work with hf and some other things, its great
